index.html

```diff
@@ -84,25 +84,35 @@ <h2 data-key="repo-id"><a href="https://www.github.com/heremaps/{repo-id}" targe
 
 		<script type="text/javascript">
 			var repos = [
-					'here-android-sdk-examples',
-					'here-ios-sdk-examples',
-					'bike-navigation',
-					'map-linkbuilder-app',
-					'postman-collections',
-					'maps-api-for-javascript-examples',
-					'jsfiddle-github',
-					'examples',
-					'dita-ot-plugins',
-					'developer-blog',
-					'data-lens-javascript-examples',
-					'here-aaa-java-sdk'
+					{id: 'here-android-sdk-examples'},
+					{id: 'here-ios-sdk-examples'},
+					{id: 'bike-navigation'},
+					{id: 'map-linkbuilder-app'},
+					{
+						id: 'postman-collections',
+						type: 'REST' // Override language value reported from API
+					},
+					{id: 'maps-api-for-javascript-examples'},
+					{id: 'jsfiddle-github'},
+					{id: 'examples'},
+					{
+						id: 'dita-ot-plugins',
+						type: 'XSLT' // Override language value reported from API
+					},
+					{id: 'developer-blog'},
+					{id: 'data-lens-javascript-examples'},
+					{id: 'here-aaa-java-sdk'}
 				],
+				repoTypeHash = {},
```

Comment:
I needed to introduce a hash for quick lookup of values, instead of filtering the `repos` array each loop of the `repos_data` to find out whether we override a repo's language or not.

Another approach would be to sanitize the `repo_data` array by filtering out irrelevant entries and overwriting / adding the language property of each record there. 

Early in this process I considered it to be an overkill (type, description, and repo id were embedded and not read via the API) so I opted for the current approach.